build/source/dshare/var_lookup.f90

```diff
@@ -144,107 +145,113 @@ integer(i4b)    :: stomResist       = 6  ! choice of function for stomatal resis
   integer(i4b)    :: newSnowDenMin             = 25  ! minimum new snow density (kg m-3)
   integer(i4b)    :: newSnowDenMult            = 26  ! multiplier for new snow density (kg m-3)
   integer(i4b)    :: newSnowDenScal            = 27  ! scaling factor for new snow density (K)
+  integer(i4b)    :: constSnowDen              = 28  ! Constant new snow density (kg m-3)
+  integer(i4b)    :: a_sn                      = 29  ! Pahaut 1976 param (kg m-3)  
+  integer(i4b)    :: b_sn                      = 30  ! Pahaut 1976 param (kg m-3 K-1)
+  integer(i4b)    :: c_sn                      = 31  ! Pahaut 1976 param (kg m-7/2 s-1/2)
+  integer(i4b)    :: d_sn                      = 32  ! Oleson et al. 2002 param (K-1)
+  integer(i4b)    :: e_sn                      = 33  ! Oleson et al. 2002 param (K)
   ! snow compaction
```

Comment:
Please use more informative parameter names. See the names that are used generally as an example. 

Also we don't use the underscore, the naming convention is `lowerCase`. See https://github.com/NCAR/summa/blob/master/docs/howto/summa_coding_conventions.md for coding conventions.